python/version_details/version_details.py

```diff
@@ -586,7 +587,7 @@ def load_data(self, entity):
         self._requested_entity = entity
 
         # If we're pinned, then we don't allow loading new entities.
-        if self._pinned and self.current_entity:
+        if self._pinned and self._current_entity:
```

Comment:
Thanks for input @eliveikis. Yes, i've searched, checked, and made changes (where necessary) to all tk components currently used by dynamite as you saw in https://github.com/shotgunsoftware/tk-multi-versiondetails/pull/22 and pointed out above.

I presume you are referring to other tk components not in use by dynamite but that still have references to .current_entity and ._request_entity. Those I have not checked. So I will ask @robblau to please give me access to all other tk components, so i can check that any references in those won't break due to this change. Thanks for bringing that up.